Application/src/controller/network/Decode.java

```diff
@@ -32,6 +32,49 @@ void execute() {
         // If no string has been allocated, abort.
         if (rawStringToDecode == null) return;
 
+        // Deployment diagram has been found.
+        if (rawStringToDecode.contains("deployment_diagram")) {
+
+            // Remove the curly brackets.
+            rawStringToDecode = removeCharactersFromString(rawStringToDecode, '{', '}', '"');
+
+            // Removes whitespaces and new lines.
+            rawStringToDecode = rawStringToDecode.replaceAll("\n", "").replaceAll("\\s", "");
+
+            // Splits the input.
+            String[] split = rawStringToDecode.split(",");
+
+            // Start on the third value (ignoring the first, second and third.)
+            for (int i = 3; i < split.length; i++) {
```

Comment:
ignore them why?

Reply:
First one is an atom for this case identifier which is already used.
Second and third and ID's which I don't need to decode it.

Reply:
+1